ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/UserManagementStore.scala

```diff
@@ -37,15 +43,40 @@ trait UserManagementStore {
     getUserInfo(id).map(_.map(_.rights))(ExecutionContext.parasitic)
   }
 
+  // TODO pbatko: test it
+  protected def decodePageToken(pageToken: String): Option[Ref.UserId] = {
+    if (pageToken.isEmpty) {
+      None
+    } else {
+      val bytes = Base64.getUrlDecoder.decode(pageToken.getBytes(StandardCharsets.UTF_8))
+      val str = new String(bytes, StandardCharsets.UTF_8)
+      Some(Ref.UserId.assertFromString(str))
+    }
+  }
```

Comment:
I'm not sure that the notion of encoded page tokens belongs to the `UserManagementStore`. The encoding choices depend on the how exactly the API is served. I'd thus suggest to move it into the `ApiUserManagementService`.

Reply:
I had this in `ApiUserManagementService` for a moment but then moved it to `UserManagementStore`. The motivation was testing that page tokens are base64 encoded rather than using plain user names.
I rejected the idea testing it in test-tool because details of the encoding used is an implementation detail and thus does not fit into conformance tests.

Reply:
Moved the code to the service layer